larva/src/main/java/nl/nn/adapterframework/testtool/TestTool.java

```diff
@@ -2857,6 +2857,14 @@ public static int executeStep(String step, Properties properties, String stepDis
 			if (fileContent == null) {
 				errorMessage("Could not read file '" + fileName + "'", writers);
 			} else {
+
+				String resolveProperties = properties.getProperty("scenario.resolveProperties");
+
+                if( resolveProperties == null || !resolveProperties.equalsIgnoreCase("false") ){
+                    AppConstants appConstants = AppConstants.getInstance();
+                    fileContent = StringResolver.substVars(fileContent, appConstants);
+                }
+
```

Comment:
Kan je niet gewoon een ignore zetten op de content die dynamisch is? 
Je kan ook replaces doen? Nu gaat de save functionaliteit kapot?

Reply:
Het gaat onder andere ook om input files, zoals voor de Navigator gebruiken we dit als input voor een adapter, waar dus een file path in staat.

Dit moet dynamisch in de XML komen te staan anders kan de adapter simpelweg niet getest worden.

`<ConvertDocument>
	<documentId>1</documentId>
	<documentLocation>${larva.staging.folder}MSG bestanden.eml</documentLocation>
	<fileName>MSG bestanden.eml</fileName>
	<GUID>ABC</GUID>
	<metaDataSchemaName/>
	<notificationAddress/>
	<documentSource>client</documentSource>
	<sourceDocumentId/>
	<sourceTransactionId/>
	<principal>KD05RX</principal>
</ConvertDocument>`

Reply:
Ok, zullen we dan doen dat de input wel dynamisch mag zijn, maar de output niet?

Reply:
Kan ik mij helemaal in vinden. 

Ik zal later vandaag even de tijd zoeken om het PR aan te passen.